README.md

```diff
@@ -10,6 +10,7 @@ To add a conference to the list, please [contribute](#contributing)!
 |------------|------|-------|-------------|
 | [JSFoo](https://jsfoo.in/2018/) | 13-14 Sep | Bengaluru | The eighth edition of India’s first JavaScript conference.
 | [PyCon India](https://in.pycon.org/2018/) | 22-26 Aug | Bengaluru International Exhibition Center (BIEC), Bengaluru | The premier conference in India on using and developing the Python programming language. |
+| [AWS Global Summits 2018](https://aws.amazon.com/summits/) | 10 May | mumbai, Mumbai | AWS GLOBAL SUMMITS are free events that bring the cloud computing community together to connect, collaborate, and learn about AWS. |
```

Comment:
Just `Mumbai` for the location.

Reply:
In the description: no need of capitalization for the conference name. Just `AWS Global Summits`. :)

Reply:
I think the link for the summit in Mumbai has not been updated yet? https://aws.amazon.com/summits/mumbai/

I'll merge this after the above changes are addressed but it would be a good idea to come back to this and update it once the above link is updated. :)

Reply:
Sure :)

Reply:
@kindacoder can you rebase and make the changes? will merge it after that.

also, looks like the link is updated.